chemie/chemie/settings/base.py

```diff
@@ -283,7 +283,7 @@
         'skin': 'bootstrapck',
         'toolbar': 'Custom',
         'toolbar_Custom': [
-            ['Bold', 'Italic', '-', 'Undo', 'Redo', '-', 'PasteText'],
+            ['Bold', 'Italic', 'Underline', 'Subscript', 'Superscript', '-', 'Undo', 'Redo', '-', 'PasteText'],
```

Comment:
I think we might restrict events ckeditor a little, since the text gets weird after we send it by email to attendees. @hambro?

Reply:
That is correct. It should be tested at least.